### DISCUSSION TITLE
Settings > Email page is throwing "Bad Gateway" errors

### DISCUSSION THREAD

**Discussion by prenaissance** [community/community#84028] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

The following page is not accessible:
![image](https://github.com/community/community/assets/95138356/6410dd25-2e75-4ed8-b27b-f90f9958002d)
Trying to navigate to it the following error page shows up with a 502 response:
![image](https://github.com/community/community/assets/95138356/daebda5a-ee0e-4c00-b238-594a95a7eb50)

**Comment by olexanderoff** [comment]
me too

**Reply by hashkode** [reply]
even got an intermittent 500 now

**Reply by R1klus** [reply]
Same here

**Reply by DaddyMadu** [reply]
same here also

**Reply by FlorianOosterhof** [reply]
Same here

**Reply by Hannibal42** [reply]
same for me

**Reply by jarvislin94** [reply]
same for me

**Reply by kaimacmaster** [reply]
same for me - raised a support ticket

**Reply by Leonardo-G** [reply]
same here

**Comment by Nikongen** [comment]
Same problem for me. 
Support:

> Thanks for contacting GitHub Support, and sorry to hear about this problem, I can confirm it's currently an ongoing issue however it's been raised with our engineering team now.

**Comment by Aldrich-chen** [comment]
![image](https://github.com/community/community/assets/155122356/b85798ae-d4de-4b30-ae17-b81486862601)

**Reply by Aldrich-chen** [reply]
是不是这个原因

**Comment by fawaf** [comment]
has this been fully fixed yet? still errors for me.
